ci/unit/docker-setup.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# This is intended to be run the plugin's root directory. `ci/unit/docker-test.sh`
+# Ensure you have Docker installed locally and set the ELASTIC_STACK_VERSION environment variable.
+set -e
+
+VERSION_URL="https://gist.githubusercontent.com/jsvd/12c60459ba0cc505dc56867561b41806/raw/f7ab015f496434cad1e97b9e4ccff0f00c31344a/versions.json"
+
+if [ "$ELASTIC_STACK_VERSION" ]; then
+    if [[ "$SNAPSHOT" = "true" ]]; then
+      ELASTIC_STACK_RETRIEVED_VERSION=$(curl $VERSION_URL -s | jq '.snapshots."'"$ELASTIC_STACK_VERSION"'"')
+      echo $ELASTIC_STACK_RETRIEVED_VERSION
+    else
+      ELASTIC_STACK_RETRIEVED_VERSION=$(curl $VERSION_URL -s | jq '.releases."'"$ELASTIC_STACK_VERSION"'"')
+    fi
+    if [[ "$ELASTIC_STACK_RETRIEVED_VERSION" != "null" ]]; then
+      # remove starting and trailing double quotes
+      ELASTIC_STACK_RETRIEVED_VERSION="${ELASTIC_STACK_RETRIEVED_VERSION%\"}"
+      ELASTIC_STACK_RETRIEVED_VERSION="${ELASTIC_STACK_RETRIEVED_VERSION#\"}"
+      echo "Translated $ELASTIC_STACK_VERSION to ${ELASTIC_STACK_RETRIEVED_VERSION}"
+      export ELASTIC_STACK_VERSION=$ELASTIC_STACK_RETRIEVED_VERSION
+    fi
+
+    echo "Testing against version: $ELASTIC_STACK_VERSION"
```

Comment:
what happens here if the `$ELASTIC_STACK_RETRIEVED_VERSION` retrieval/parsing failed? the `travis.yml` value like `6.x` will not be valid?

Reply:
I mean, will not be valid propagted downstream into `Dockerfile` ?